src/kernels/float_types.h

```diff
@@ -203,11 +203,11 @@
 
 #if MIOPEN_USE_BFP16 == 1
 #ifdef __HIP_PLATFORM_AMD__
-#define CVT_FLOAT2ACCUM(x) MIOPEN_ERROR_NOT_IMLEMENTED
-#define CVT_ACCUM2FLOAT(x) MIOPEN_ERROR_NOT_IMLEMENTED
-#define CVT_INTEGRAL2ACCUM(x) MIOPEN_ERROR_NOT_IMLEMENTED
-#define CVT_FP32_2FLOAT(x) MIOPEN_ERROR_NOT_IMLEMENTED
-#define CVT_FP32_2ACCUM(x) MIOPEN_ERROR_NOT_IMLEMENTED
+#define CVT_FLOAT2ACCUM(x) (bfloat16_to_float(x))
+#define CVT_ACCUM2FLOAT(x) (float_to_bfloat16(x))
+#define CVT_INTEGRAL2ACCUM(x) (static_cast<_FLOAT_ACCUM>(x))
```

Comment:
as suggested at https://github.com/ROCm/MIOpen/pull/2623#discussion_r1450656051, it should be:
```suggestion
#define CVT_INTEGRAL2ACCUM(x) (static_cast<FLOAT_ACCUM>(x))
```

Reply:
Fixed as per your comment.